extensions/fs/test_fs.lua

```diff
@@ -0,0 +1,331 @@
+-- hs.fs tests
+
+homeDir = os.getenv("HOME").."/"
```

Comment:
I'm a little nervous about depending on the environment for a path, and then appending "/" to it, it has all the makings of a "rm -r /" bug.

How about we just hard code to using `/tmp/hs_fs_test_dir/`?